deploy/lib/ml_rest.rb

```diff
@@ -306,7 +274,7 @@ def install_transforms(path)
           @logger.debug "transformName: #{transformName}"
           @logger.debug "headers: #{headers}"
           @logger.debug "params: #{params}"
-          url = "#{@rest_protocol}://#{@hostname}:#{@port}/v1/config/transforms/#{transformName}"
+          url = "http://#{@hostname}:#{@port}/v1/config/transforms/#{transformName}"
```

Comment:
Thank Dave - Yes, it looks like my fork was out of sync. I'll investigate when I get a chance.